src/core/tour/TourManager.ttslua

```diff
@@ -19,6 +20,19 @@ do
     z = 0,
   }
 
+  -- Global XML coordinates where we can present a card
+  local SCREEN_POSITIONS = {
+    center = "0 0 0",
+    north = "0 300 0",
+    east = "600 0 0",
+    west = "-600 0 0",
+    south = "0 -300 0",
+    northwest = "-600 300 0",
+    northeast = "600 300 0",
+    southwest = "-600 -300 0",
+    southeast = "600 -300 0"
```

Comment:
Are these values universally applicable or dependant on screen resolution?

Reply:
I'm honestly not sure.  Will have to test this more to see.

Reply:
There's still going to be a few more passes on this, including testing, so would prefer not to block this one at the moment.